apps/aetx/src/aetx_sign.erl

```diff
@@ -56,7 +59,8 @@ sign(Tx, PrivKey) when is_binary(PrivKey) ->
   sign(Tx, [PrivKey]);
 sign(Tx, PrivKeys) when is_list(PrivKeys) ->
     Bin = aetx:serialize_to_binary(Tx),
-    Signatures = [ enacl:sign_detached(Bin, PrivKey) || PrivKey <- PrivKeys ],
+    Signatures = [ enacl:sign_detached(Bin, PrivKey)
+                   || PrivKey <- PrivKeys, ?VALID_PRIVK(PrivKey)],
```

Comment:
This is silently filtering. May I suggest an explicit lists:filter if this is intention?

Reply:
I doubt you want to filter here, rather fail in a controlled way?